addons/l10n_ug/data/account_tax_report_data.xml

```diff
@@ -514,21 +514,49 @@
                             <record id="F31_normal_tax" model="account.report.expression">
                                 <field name="label">hidden_computation_of_F31_using_normal_formula</field>
                                 <field name="engine">aggregation</field>
-                                <field name="formula">
-                                    (UG_TAX_D20.tax - UG_TAX_F30.disallowed)
-                                    * (UG_TAX_C1.base + UG_TAX_C2.base + UG_TAX_C4.base + UG_TAX_C5.base + UG_TAX_C6.base + UG_TAX_C7.base)
-                                    / (UG_TAX_C1.base + UG_TAX_C2.base + UG_TAX_C3.base + UG_TAX_C4.base + UG_TAX_C5.base + UG_TAX_C6.base + UG_TAX_C7.base)
-                                </field>
+                                <field name="formula">UG_TAX_F31.normal_formula_part1 * UG_TAX_F31.normal_formula_part2 / UG_TAX_F31.normal_formula_part3</field>
+                                <field name="subformula">ignore_zero_division</field>
+                            </record>
+                            <record id="F31_normal_tax_part1" model="account.report.expression">
+                                <field name="label">normal_formula_part1</field>
```

Comment:
maybe the name could be changed, normal_formula doesn't mean anything

Reply:
i think its fine like this because the UG_TAX_F31.allowed value is calculated using two hidden formulas called the normal formula and the sam formula, and i just split both of them into parts

Reply:
Yeah i found the name not very clear but it was already name like this hidden_computation_of_F31_using_normal_formula so let's keep what you added :smile: